common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -122,11 +195,15 @@ function() {
             this.video.removeEventListener('pause', this.onPause, false);
             this.video.removeEventListener('ended', this.onEnded, false);
             this.el
-                .find('.video-player div').removeClass('is-hidden')
+                .find('.video-player div')
+                .removeClass('is-hidden')
                 .end()
-                .find('.video-player .video-error').addClass('is-hidden')
-                .end().removeClass('is-initialized')
-                .find('.spinner').attr({'aria-hidden': 'false'});
+                .find('.video-player .video-error')
```

Comment:
We can do same thing here as mentioned above to have a `.video-player' element and do following calls on it